app/views/layouts/course.html.slim

```diff
@@ -25,4 +25,5 @@
     div.col-lg-10.col-md-9.col-sm-8 class=page_class
       = yield
 
-    #popup-notifier data=controller.next_popup_notification
+    - if current_course_user && UserNotification.next_unread_popup_for(current_course_user).present?
```

Comment:
Actually a small note - `current_course_user` can be `current_course_user&.student?`

Reply:
I don't think it's very robust. It is not unlikely that we might have `UserNotification` popups for staff. Prefer not to add it, but it may save some clock cycles.

If we go ahead, would an ability check (maybe `can? :create, UserNotification` or `:show`) be more appropriate?

Reply:
The ability check does an SQL call, if that is the case might as well do the `#next_unread_popup_for` instead.

If that is the case, suggest we leave it as is for now then!